## Bioinformatics_resources
This repository containÂ *resources* that I found useful on my path to **mastering Bioinformatics**. There are links to proper sites with proper documentation of concepts, books, and other sources of information.

#### An array of Bioinformatics techniques to grasp
-[R amrkdown](https://ourcodingclub.github.io/tutorials/rmarkdown/#what) - A proper introduction to R markdown with examples
- [DGE analysis](http://bioconductor.org/packages/3.7/workflows/vignettes/RnaSeqGeneEdgeRQL/inst/doc/edgeRQL.html#design-matrix) - End to end analysis of Differentially expressed genes
- [Rmd guide](https://www.markdownguide.org/basic-syntax/) - Concepts of Git and Github
- [Prevent the terminal from logging off the server](https://linuxhandbook.com/ssh-broken-pipe-error/) - Practical solution
- [Single cell RNA analysis](https://hbctraining.github.io/scRNA-seq_online/schedule/links-to-lessons.html) - Single cell RNASeq analysis
- [All about linux](http://www.ee.surrey.ac.uk/Teaching/Unix/) - wonderful linux resource
- [Rmarkdown cheat sheet](https://www.rstudio.com/wp-content/uploads/2015/02/rmarkdown-cheatsheet.pdf) - A summary to how R markdown works
- [GWENA:](https://github.com/Kumquatum/GWENA)- An Awesome pipeline to construct a Biological network, Module construction and Enrichment analysis
- [WCL](http://www.commonwl.org/user_guide/01-introduction/index.html) - A guide for efficient pipeline writing and sharing
- [How to use Sratoolkit](https://www.reneshbedre.com/blog/ncbi_sra_toolkit.html) - A guidance to effectively using SRAtool
- [Using du and df](https://www.pair.com/support/kb/paircloud-du-and-df/) - These are important commands to know disk utilization and storage space use. The commands are particularly important when using HPC resources
- [Using R](https://swcarpentry.github.io/r-novice-inflammation/aio/index.html) - A good R learning platform
- [R for data sceince](https://r4ds.had.co.nz/r-markdown.html)
- [Differential gene expression](https://hbctraining.github.io/Training-modules/planning_successful_rnaseq/lessons/sample_level_QC.html#:~:text=First%2C%20the%20count%20data%20needs,using%20your%20tool%20of%20interest.) - Analysis steps to counduct differential gene expression of RNA-Seq data
- [Differential gene expression](https://github.com/hbctraining/DGE_workshop) - A systematic guide to Differential gene expression
- [Havard bioinformatics core](https://github.com/hbctraining) - Bioinformatics training by harvard bioinformatics core
- [Sample work of WGCNA](https://www.polarmicrobes.org/weighted-gene-correlation-network-analysis-wgcna-applied-to-microbial-communities/) - An example of analysis pipeline with code and results obtained.
- [stable UCSC human reference genome](https://www.gungorbudak.com/blog/2018/05/16/how-to-download-hg38-grch38-fasta-human-reference-genome/) - Download the stable reference hg38 genome here
- [gene code reference genome](https://www.gencodegenes.org/human/) - Get both hg18 and hg19 reference genomes here
- [Next generation sequence analysis](https://hbctraining.github.io/main/) - A comprehensive NGS series analysis
- [Hisat2 tutorial](https://bioinformatics-core-shared-training.github.io/RNAseq_September_2019/html/C_Alignment_with_HISAT2_practical.html) - A good tutorial of using hisat2
#### WGCNA papers
- [DGE + Network analysis](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6788446/) - This paper combines network analysis and differential gene expression. It includes actual codes used in analysis pipeline.
- [WGCNA and DGE analysis](https://journals.lww.com/md-journal/fulltext/2019/09130/using_weighted_gene_co_expression_network_analysis.30.aspx)- This paper has the pipeline to conduct DGE and network analysis.
- [T-mux tutorial](https://github.com/tmux/tmux/wiki/Getting-Started) - An excellent guide to Tmux usage

#### Breast cancer papers
- [Network analysis paper](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6688090/) - This is aa well curated paper that describes Gene co-expression analysis

#### Contaminating DNA Project
- [viGEN pipeline](https://github.com/Jkgitau/viGEN) - An elaborate methodology used to conduct this study. All the code, tools and exaplanations are provided
- [Decontaminer github](https://github.com/amarinderthind/decontaminer) -  Analysis steps of decontaminer tool - Analysis steps of decontaminer tool -  RNA-Seq contamination project
- [Decontaminer publication](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-019-2684-x) - A new pipeline that detects viruses, bacteria and fungi contaminants

#### You tube channels
